src/registrar/views/portfolios.py

```diff
@@ -22,7 +22,10 @@ class PortfolioDomainsView(PortfolioDomainsPermissionView, View):
     template_name = "portfolio_domains.html"
 
     def get(self, request):
-        return render(request, "portfolio_domains.html")
+        context = {}
+        if self.request and self.request.user and self.request.user.is_authenticated:
```

Comment:
I wonder if the ? operator would work here...

Reply:
I believe it's possible to do something like `context = {"user_domain_count": self.request...} if self.request and self.request.user and self.request.user.is_authenticated else {}` but I think since we're working with so many conditionals Zander's code is much cleaner and readable